src/schemathesis/_hypothesis.py

```diff
@@ -24,11 +24,17 @@
 
 
 def create_test(
-    endpoint: Endpoint, test: Callable, settings: Optional[hypothesis.settings] = None, seed: Optional[int] = None
+    endpoint: Endpoint,
+    test: Callable,
+    settings: Optional[hypothesis.settings] = None,
+    seed: Optional[int] = None,
+    before_one_hook: Optional[Callable] = None,
 ) -> Callable:
     """Create a Hypothesis test."""
     hook_dispatcher = getattr(test, "_schemathesis_hooks", None)
     strategy = endpoint.as_strategy(hooks=hook_dispatcher)
+    if before_one_hook:
```

Comment:
I believe, that we can use schema- and test-local hooks as well here if this function will be a generator